ChangeFeedLabSolution/Startup.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Azure.Messaging.EventHubs.Producer;
+using Microsoft.Azure.Functions.Extensions.DependencyInjection;
+using Microsoft.Extensions.Azure;
+
+[assembly: FunctionsStartup(typeof(ChangeFeedFunction.Startup))]
+namespace ChangeFeedFunction
+{
+    public class Startup : FunctionsStartup
+    {
+        public const string EventHubName = "event-hub1";
```

Comment:
This should read from configuration rather than being hardcoded here.

Reply:
@jsquire - Updated.